app/src/main/java/com/palone/planahead/screens/taskEdit/data/AlertFieldProperty.kt

```diff
@@ -0,0 +1,10 @@
+package com.palone.planahead.screens.taskEdit.data
+
+import com.palone.planahead.data.database.alert.properties.AlertType
+import java.time.temporal.ChronoUnit
+
+data class AlertFieldProperty(
+    val type: AlertType = AlertType.NOTIFICATION,
+    val valueToShowAlertBeforeTaskEvent: Int = 1,
```

Comment:
Unclear variable name. I have no idea what this is.

Reply:
yeah, it should be called offsetValue i think